docs/standard/microservices-architecture/multi-container-microservice-net-applications/integration-event-based-microservice-communications.md

```diff
@@ -74,7 +76,7 @@ In the [Observer pattern](https://en.wikipedia.org/wiki/Observer_pattern), your
 
 ### Publish-subscribe (Pub/Sub) pattern 
 
-The purpose of the [Pub/Sub pattern](https://msdn.microsoft.com/en-us/library/ff649664.aspx) is the same as the Observer pattern: you want to notify other services when certain events take place. But there is an important semantic difference between the Observer and Pub/Sub patterns. In the Pub/Sub pattern, the focus is on broadcasting messages. In contrast, in the Observer pattern, the Observable does not know who the events are going to, just that they have gone out. In other words, the Observable (the publisher) does not know who the Observers (the subscribers) are.
+The purpose of the [Pub/Sub pattern](https://msdn.microsoft.com/en-us/library/ff649664.aspx) is the same as the Observer pattern: you want to notify other services when certain events take place. But there is an important semantic difference between the Observer and Pub/Sub patterns. In the Pub/Sub pattern, the focus is on broadcasting messages. In addition, in the Observer pattern, the Observable does not know who the events are going to, just that they have gone out. In other words, the Observable (the publisher) does not know who the Observers (the subscribers) are.
```

Comment:
This paragraph is very confusing.  "There is an important semantic difference..." implies you're about to tell the reader something important about the differences.  Then, you say that Pub/Sub emphasizes "broadcasting messages". To whom? That's apparently not important. Then, you tell the reader that the important part of the  Observer pattern  is that the Observable doesn't know who the events are going to. That sounds a lot like a broadcast protocol. 

The reader is going to be left very confused as to what the important semantic difference is.

Reply:
I changed this so it is clearer. In fact, there was a typo in the narrative where it was saying the contrary...
This is it:
"But there is an important difference between the Observer and Pub/Sub patterns. In the observer pattern, the broadcast is performed directly from the observable to the observers, so they “know” each other. But when using a Pub/Sub pattern, there is a third component, called broker or message broker or event bus, which is known by both the publisher and subscriber. Therefore, when using the Pub/Sub pattern the publisher and the subscribers are precisely decoupled thanks to the mentioned event bus or message broker."